core/src/main/java/nl/nn/adapterframework/jdbc/migration/LiquibaseResourceAccessor.java

```diff
@@ -69,8 +72,9 @@ public InputStreamList openStreams(String relativeTo, String streamPath) throws
 	public InputStream openStream(String relativeTo, String path) throws IOException {
 		if(path.equals(resource.getSystemId())) {
 			return resource.openStream();
+		} else if(classLoader != null) { // script may contain include(s)
+			return ((ClassLoaderBase)classLoader).getResource(path, false).openStream();
```

Comment:
I'm heavily leaning towards using `ClassUtils.getResourceURL(resource, path).openStream();`
(You'll have to add the correct classloader/scopeProvider to the bytesresources)

Reply:
I agree with Niels